### ISSUE TITLE
Event dialogue window formatting

### ISSUE THREAD

**Issue by GrantSP** [LoneGazebo/Community-Patch-DLL#2508]
_1. Mod Version (i.e Date - 4/23):_
September 12th

_2. Mod List (if using Vox Populi only, leave blank):_

_3. Error Description:_ 
The new Events dialogue window shows a vertical scrollbar on the left side even when nothing is on the selected tab and it also is longer than the actual dialogue.
My screen resolution is 1920 x 1080 if that makes any difference.

_4. Steps to reproduce (optional):_

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):

2. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

3. Screenshots (optional):

**Comment by LoneGazebo** [comment]
That's intentional - fills the space and lets users know that they can scroll.

**Comment by GrantSP** [comment]
You may have intended to always have a scrollbar but that's not the issue, the scrollbar is too long for the actual dialogue. It extends for about 10% longer than the dialogue it is on. Of course the actual measurements will be different I assume on different resolutions.
![events overview](https://cloud.githubusercontent.com/assets/3291695/18461580/81949ad4-79be-11e6-95d4-6c90f44a7322.png)

**Comment by LoneGazebo** [comment]
Oh, that's odd, I don't see that issue on my end. I'll check.

**Comment by GrantSP** [comment]
Sorry, but it is still present. To add to the matter it now renders the 'End turn' button useless if you open the dialogue and then close it. For some reason the button, which will be labelled: 'Unit needs orders' refuses to cycle through your units and you are never able to get to the 'End Turn'.

**Comment by LoneGazebo** [comment]
Accidentally closed. Haven't looked at yet. 

Has no bearing on end turn button, though. It can't.

**Comment by GrantSP** [comment]
If there is something in the Lua that is grabbing the mouseclick function that may be the issue?

**Comment by LoneGazebo** [comment]
secondary issue is an unconnected ilteroi pathfinder bug

**Comment by GrantSP** [comment]
That's good! 1 down, 1 to go.

**Comment by LoneGazebo** [comment]
If you want to do some debugging, the offending line is

<ScrollBar Style="VertSlider" Length="550" Offset="0,18" Anchor="L,T" AnchorSide="I,I"/>

in CityRecentEventInfos.xml, PlayerEventsInfo.xml, CityEventsInfo.xml and RecentEventsInfo.xml

(in CoreChanges/CoreLua)

Set the Length to 400 or something.

**Comment by GrantSP** [comment]
What did you fix? I'm still trying to figure out what is wrong?

**Comment by LoneGazebo** [comment]
Reduced size down to 360. It'll be in the next update, so don't worry about it. :)

**Comment by GrantSP** [comment]
But that's the problem. I've already changed those figures in my files to many different values and it does not change.
Just to be sure I was actually making changes that are reflected in the game I've switched it to the right hand side and still the actual scrollbar length is too long.
The bottom anchor gets positioned correctly but the actual scrollbar overlaps it. It shouldn't happen, I thought those UI elements only show when necessary and are dynamically sized.
